libs/service-portal/information/src/components/FamilyMemberCard/FamilyMemberCard.tsx

```diff
@@ -8,6 +8,7 @@ import { InformationPaths } from '../../lib/paths'
 interface Props {
   title: string
   nationalId: string
+  baseId?: string
```

Comment:
What does it mean if `baseId` is `undefined`?

Reply:
Not all every familymember has a baseId. Like for example the logged in user and spouse.